haigha2/transports/eventlet_transport.py

```diff
@@ -48,12 +50,11 @@ def __init__(self, *args, **kwargs):
     # Transport API
     ###
 
-    def connect(self, address):
+    def connect(self, host_port_tuple):
```

Comment:
why we renamed the argument to `host_port_tuple`? 
the `connect` interface useually pass address and lets the underline layer to handle the address format
https://docs.python.org/2/library/socket.html#socket.socket.connect